src/components/PDF/ApplicationForm.tsx

```diff
@@ -182,6 +182,16 @@ class ApplicationForm extends Component<Props, State> {
     }
   }
 
+  progressBarFill() {
```

Comment:
arrow function (so do progressBarFill = () => {}) and also add a type declaration. This returns an int right? so lets define the return as an int